README.md

```diff
@@ -1,32 +1,30 @@
 # Hyperon DAS
 
-Hi! This package is a query engine API for Distributed AtomSpace (DAS). When is possible execute queries using **[Pattern Matcher](https://wiki.opencog.org/w/The_Pattern_Matcher)**
+A data manipulation API for Distributed Atomspace (DAS). It allows queries with pattern matching capabilities and traversal of the Atomspace hypergraph.
 
 ## Table of Contents
 - [Hyperon DAS](#hyperon-das)
   - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
   - [Installation](#installation)
     - [Using-pip](#using-pip)
     - [Using-Poetry](#using-poetry)
-  - [Pre-Commit Setup](#pre-commit-setup)
   - [Usage](#usage)
-    - [Local](#local)
-    - [Remote](#remote)
-    - [Server](#server)
+    - [Local DAS](#local-das)
+    - [Remote DAS](#remote-das)
+    - [Server DAS](#server-das)
   - [TraverseEngine](#traverseengine)
     - [Creating a TraverseEngine object](#creating-a-traverseengine-object)
     - [Traversal Methods](#traversal-methods)
     - [Parameters for Traversal Methods](#parameters-for-traversal-methods)
   - [Examples](#examples)
-    - [Local DAS](#local-das)
-    - [Remote DAS](#remote-das)
-      - [Local Scope](#local-scope)
-      - [Remote scope](#remote-scope)
-      - [Remote scope synchronized with local Atoms](#remote-scope-synchronized-with-local-atoms)
-    - [Traverse](#traverse)
   - [Tests](#tests)
   - [Release Notes](#release-notes)
 
+## Overview
```

Comment:
Review topics structure after removing sections to avoid having a lot of small sections. Some information may be better structure as bullet point lists.